docs/standard/garbage-collection/performance.md

```diff
@@ -45,8 +45,6 @@ ms.workload:
 ### Debugging with SOS  
  You can use the [Windows Debugger (WinDbg)](/windows-hardware/drivers/debugger/index) to inspect objects on the managed heap.  
   
- To install WinDbg, install Debugging Tools for Windows from the [WDK and Developer Tools Web site](http://go.microsoft.com/fwlink/?LinkID=103787).  
```

Comment:
This link was broken and the page linked in the sentence above contains installation notes.

Reply:
I think we could link to here: https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools

Reply:
@mairaw that's true; fixed!